app.js

```diff
@@ -127,17 +141,5 @@ var s3 = new AWS.S3();
 
 startRequests();
 
-setInterval(function () {
-  var uptime = process.uptime();
-  console.log({
-    uptime: uptime,
-    listBufferSize: objectsToCopy.length,
-    succeeded: progress.succeeded,
-    failed: progress.failed,
-    gigabytesTransferred: progress.bytesTransferred/1024/1024/1024,
-    megabitsPerSecond: ((progress.bytesTransferred/1024/1024) * 8) / uptime,
-    objectsPerSecond: progress.succeeded / uptime,
-    marker: nextMarker
-  });
-
-}, 1000);
+setInterval(outputStatus, 1000).unref();
```

Comment:
There could be multiple instances, but that doesn't have any bearing on this issue.  Also, there is only one timer and the documentation comment is referring to `setTimeout`, but I'm using `setInterval`.